docs/publishing.md

```diff
@@ -7,12 +7,11 @@ Some guidelines:
 1. We publish from the master branch.
 1. We specify a specific tag that matches the Salesforce CLI version for each
    image.
-1. We also have a `latest` tag that matches the `latest` tag on the Salesforce
+1. We also have a `latest-slim` and `latest-full` tags that matches for the latest version of the Salesforce
```

Comment:
Maybe we should say the tag matches _with_ the latest version of the CLI instead of _for_?
```suggestion
1. We also have a `latest-slim` and `latest-full` tags that matches with the latest version of the Salesforce
```